CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ created.
 ## Upcoming
 
 ### Added
+- ([#667](https://github.com/microsoft/InnerEye-DeepLearning/pull/667)) Automatically and linearly scale the learning rate of the SSL encoder to the number of GPUs.
```

Comment:
Unrelated question: why do we add the link manually if GitHub does it automatically?